filters/filter_17_TrackParam/template.txt

```diff
@@ -2,14 +2,14 @@
 ! License: https://github.com/AdguardTeam/AdguardFilters/blob/master/LICENSE
 !
 !---------------------------------------------------------------------------!
-!-------------- General tracking parameters list  ----------------------------------!
+!-------------- General tracking parameters list  --------------------------!
 !---------------------------------------------------------------------------!
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/TrackParamFilter/sections/general_url.txt"
 !---------------------------------------------------------------------------!
-!-------------- Specific web sites ---------------------------------------------!
+!-------------- Specific web sites -----------------------------------------!
 !---------------------------------------------------------------------------!
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/TrackParamFilter/sections/specific.txt"
 !---------------------------------------------------------------------------!
-!-------------- Whitelist ------------------------------------------------------!
+!-------------- Whitelist --------------------------------------------------!
```

Comment:
It looks like that there is one more to change.

Reply:
Yep, I saw that. I'm doing this now.